### ISSUE TITLE
Arabic MFA Dictionary Unavailable on Documentation Link

### ISSUE THREAD

**Issue by khelfiakram1** [MontrealCorpusTools/mfa-models#35]
When trying to install the Arabic MFA dictionary using the command: `mfa model download dictionary arabic_mfa` I encounter the following error: "Could not find a model named "arabic_mfa" for dictionary." even the link to the Arabic MFA Dictionary provided on the documentation page is unavailable or broken.

**Comment by zelalemamera-stonybrook** [comment]
yes, It seems like the dictionary is not available, even though it is listed on the website. Does anyone know where I can obtain a dictionary from Arabic to MFA or some other phone set?
